Assignment 2.Rmd

```diff
@@ -87,14 +91,100 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
+teachers=data.frame(EDGE %>%
+                      group_by(from) %>%
+                      summarise(count = n()))
+teachers
+
+students<-count(group_by(EDGE,to))
+students
+
+#students<-summarise(group_by(EDGE,to),count=n())
+
+names(teachers)=c("characters","count")
+teachers
+
+names(students)=c("characters","count")
+students
+
+VERTEX2<-dplyr::bind_rows(teachers,students)
+VERTEX2
+
+library(igraph)
+
+g2=graph.data.frame(EDGE, directed=TRUE, vertices=VERTEX2)
+g2
+plot(g2,layout=layout.fruchterman.reingold, vertex.color = VERTEX$gender, vertex.size = (VERTEX2$count)*5)
+
```

Comment:
Good work sizing the vertices, you may want to use `log()` instead of `x 5` so that the teacher vertices are not so massive though. Also, you may want to shrink the arrow heads.